Example-Integrating-Network-diffeqpy.ipynb

```diff
@@ -0,0 +1,415 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "25a2c213-0309-4db7-8cfd-729e7461afdb",
+   "metadata": {},
+   "source": [
+    "# **Example of Integrating a Network**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5af9a0c8-6ded-47af-8d71-6c6a83631e86",
+   "metadata": {},
+   "source": [
+    "## Creating a Network"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e3d5dfa6-954c-4969-b7b7-aa260d1d0b5e",
+   "metadata": {},
+   "source": [
+    "We will start by looking at the CNO network for simplicity. We will first need to import pynucastro, read through the ReacLib library, and pass in the names of the reaction rates we are interested in. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "65a170c4-1f91-4394-a319-43573a1bb68b",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[C12 + p ⟶ N13 + 𝛾,\n",
+       " C13 + p ⟶ N14 + 𝛾,\n",
+       " N13 ⟶ C13 + e⁺ + 𝜈,\n",
+       " N13 + p ⟶ O14 + 𝛾,\n",
+       " N15 + p ⟶ He4 + C12,\n",
+       " O14 ⟶ N14 + e⁺ + 𝜈,\n",
+       " O15 ⟶ N15 + e⁺ + 𝜈]"
+      ]
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import pynucastro as pyna\n",
+    "rl = pyna.ReacLibLibrary()\n",
+    "rate_names = [\"c12(p,g)n13\",\n",
+    "              \"c13(p,g)n14\",\n",
+    "              \"n13(,)c13\",\n",
+    "              \"n13(p,g)o14\",\n",
+    "              \"n15(p,a)c12\",\n",
+    "              \"o14(,)n14\",\n",
+    "              \"o15(,)n15\"]\n",
+    "\n",
+    "rates = rl.get_rate_by_name(rate_names)\n",
+    "rates"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ff5544a8-939a-4078-8aeb-5446cd1ec1b1",
+   "metadata": {},
+   "source": [
+    "<font color= blue>PythonNetwork</font> has methods to write thhe RHS of the system of ODEs."
```

Comment:
You should write `` `PythonNetwork` `` (surrounded by backticks) instead of `<font color= blue>PythonNetwork</font>` for in-line code